integration/internal/emulator/blockchain.go

```diff
@@ -1034,6 +1045,7 @@ func (b *Blockchain) executeSystemChunkTransaction() error {
 		fvm.WithSequenceNumberCheckAndIncrementEnabled(false),
 		fvm.WithRandomSourceHistoryCallAllowed(true),
 		fvm.WithBlockHeader(b.pendingBlock.Block().Header),
+		fvm.WithAccountStorageLimit(false),
```

Comment:
@turbolent I added this for feature parity.

Reply:
Great, thank you!